src/picologging/__init__.pyi

```diff
@@ -131,7 +131,7 @@ class Handler(Filterer):
     def createLock(self) -> None: ...
 
 class Logger(Filterer):
-    propagate: bool
+    propagate: object
```

Comment:
Object is the wrong type to use here (`Any` is for anything, not object), but we want to be specific that it has the `__bool__` magic. The Python stdlib should have a `SupportsBool` meta type, but it doesn't so here is one:

```python
from typing import runtime_checkable, Protocol

@runtime_checkable
class SupportsBool(Protocol):
    """An ABC with one abstract method __bool__."""
    __slots__ = ()

    @abstractmethod
    def __bool__(self) -> bool:
        pass
```

It might be worth contributing this to the `typing` module as a PR, I don't know why its missing.

Reply:
Thanks for clarifying, just read more in the docs.
Submitted PR https://github.com/python/cpython/pull/97570
I'll add it locally for now.